### PR TITLE
Docker build simplification

### PR DESCRIPTION
**Pull request by davidslater**

### CODE REVIEW

File: armory/__init__.py [hunk 1]
```diff
@@ -6,6 +6,11 @@
 logger = logging.getLogger(__name__)
 logger.addHandler(logging.NullHandler())
 
+
+# Semantic Version
+__version__ = "0.13.0"
```
Comment (davidslater) [inline armory/__init__.py:11]: Since `armory.docker.images` references the semantic version, I wanted this to proceed before importing.

**Comment by davidslater** [comment]
Tests running here: https://github.com/davidslater/armory/actions/runs/637617590

**Comment by davidslater** [comment]
Tests failed due to versions. Updated tests running here: https://github.com/davidslater/armory/actions/runs/637734285

File: tools/update_config_version.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
+"""
+Script to find and update all armory versions in JSON configs to current version
```
Comment (mwartell) [inline tools/update_config_version.py:2]: In reviewing this pull request, I thought "gee, hitting all those json files is tedious and error prone. I should have expected the existence of this script based on the PR author.

**Review by mwartell — APPROVED** [review]
Sprawling but thorough. LGTM.

**Pull request merged**